MAUI/Picker/populating-items.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: post
+title: Populating with .NET MAUI Picker control | Syncfusion
+description: Learn here all about populating items with Syncfusion .NET MAUI Picker (SfPicker) control.
+platform: maui
+control: SfPicker
+documentation: ug
+---
+
+# Populating Items in MAUI Picker (SfPicker)
+
+In .NET MAUI, you can bind a Picker control to an external data source by setting its ItemsSource property to an ObservableCollection<object>. This allows the Picker to display and update data from the collection.
```

Comment:
In .NET MAUI, you can bind a Picker control to an external data source by setting its ItemsSource property to an ObservableCollection<object>. This enables the Picker to display and update data from the collection.

Reply:
Content modified